src/applications/hca/config/form.js

```diff
@@ -85,14 +85,7 @@ import vaFacilityApiPage from './chapters/insuranceInformation/vaFacility_api';
 const { dependents: DEPENDENT_PATHS } = SHARED_PATHS;
 
 // declare schema definitions
-const {
-  date,
-  fullName,
-  monetaryValue,
-  phone,
-  provider,
-  ssn,
-} = fullSchemaHca.definitions;
+const { date } = fullSchemaHca.definitions;
 const dependentSchema = createDependentSchema(fullSchemaHca);
```

Comment:
This is going to break since there is no definition for dependent anymore. Go through that definition file and see what you can grab to fix that.